ctapipe/io/containers.py

```diff
@@ -763,6 +768,12 @@ class WaveformCalibrationContainer(Container):
         "np array of (digital count) to (photon electron) coefficients (n_chan, n_pix)"
     )
 
+    pedestal_per_sample = Field(
```

Comment:
Is this pedestal for each sample? Or pedestale averaged over samples?

If it is per sample, I would expect shape `(n_channel, n_pixel, n_samples)`

Reply:
It is an averaged value but corresponding to one sample, to put it the same value for n_sample it seems to me overdone... also because generally this value is subtracted from the charge. I put the value for one sample because the signal integration window could be different from the pedestal estimation window, so this value will be simply multiplied by the signal window width...

Reply:
Ah ok, understood.